helloWorld.py

```diff
@@ -10,20 +13,27 @@ def recurseForUsers(hiya):
 
 if __name__ == "__main__":
 
-	greeting = "Hello World"
-
+	# Let the user customize their greeting; it's all about personal choice
+	if len(sys.argv) > 1:
```

Comment:
good to see you putting in a default in case of no user input, but for future work have you considered adding command line help or even integrating with an existing cli library like https://click.palletsprojects.com/en/8.1.x/ ?

Reply:
This suggestion has been attached to a sticky note on a whiteboard and will be added to the Jira as soon as we get a Jira.